build/template-Build-run-tests-sign.yml

```diff
@@ -0,0 +1,213 @@
+    TargetFolder: '$(Build.ArtifactStagingDirectory)\ProductBinaries'
+
+- task: securedevelopmentteam.vss-secure-development-tools.build-task-policheck.PoliCheck@2
+  displayName: 'Run PoliCheck'
+  inputs:
+    targetType: F
+    result: PoliCheck.xml
+    optionsFC: 0
+    optionsXS: 0
+    optionsHMENABLE: 0
+  condition: and(succeeded(), eq(variables['PipelineType'], 'legacy'))
+
+- task: securedevelopmentteam.vss-secure-development-tools.build-task-credscan.CredScan@3
+  displayName: 'Run CredScan'
+  inputs:
+    outputFormat: pre
+    suppressionsFile: 'build/credscan-exclusion.json'
+    debugMode: false
+  condition: and(succeeded(), eq(variables['PipelineType'], 'legacy'))
+
+- task: securedevelopmentteam.vss-secure-development-tools.build-task-roslynanalyzers.RoslynAnalyzers@2
+  displayName: 'Run Roslyn Analyzers'
+  condition: and(eq(variables['TargetNet8'], 'False'), eq(variables['PipelineType'], 'Legacy'))
+
+- task: securedevelopmentteam.vss-secure-development-tools.build-task-binskim.BinSkim@4
+  displayName: 'Run BinSkim'
+  inputs:
+    AnalyzeSymPath: '$(Build.ArtifactStagingDirectory)\ProductBinaries'
+    AnalyzeVerbose: true
+    AnalyzeHashes: true
+  condition: and(succeeded(), eq(variables['PipelineType'], 'legacy'))
+
+- task: securedevelopmentteam.vss-secure-development-tools.build-task-publishsecurityanalysislogs.PublishSecurityAnalysisLogs@3
+  displayName: 'Publish Security Analysis Logs'
+  continueOnError: true
+
+- task: securedevelopmentteam.vss-secure-development-tools.build-task-postanalysis.PostAnalysis@2
+  displayName: 'Post SDL Analysis'
+  continueOnError: true
+
+#Sign Wilson 7x task group
+- template: template-sign-wilson.yaml
+
+- task: DotNetCoreCLI@2
+  displayName: 'Pack libraries'
+  inputs:
+    command: 'pack'
+    arguments: '--no-restore'
+    packDirectory: '$(Build.SourcesDirectory)\artifacts'
+    nobuild: true
+    verbosityPack: 'Minimal'
+    configuration: $(BuildConfiguration)
+    packagesToPack: '$(WilsonSourceDirectory)Product.proj'
+    feedsToUse: 'config'
+    nugetConfigPath: Nuget.config
+    externalFeedCredentials: 'Internal Analyzers'
+
+- task: EsrpCodeSigning@4
+  displayName: 'Sign Nuget Packages'
+  inputs:
+    ConnectedServiceName: 'IDDP Code Signing'
+    FolderPath: artifacts
+    Pattern: '*.nupkg'
+    signConfigType: inlineSignParams
+    inlineOperation: |
+      [
+          {
+              "keyCode": "CP-401405",
+              "operationSetCode": "NuGetSign",
+              "parameters": [ ],
+              "toolName": "sign",
+              "toolVersion": "1.0"
+          },
+          {
+              "keyCode": "CP-401405",
+              "operationSetCode": "NuGetVerify",
+              "parameters": [ ],
+              "toolName": "sign",
+              "toolVersion": "1.0"
+          }
+      ]
+    SessionTimeout: 20
+  condition: and(succeeded(), eq(variables['PipelineType'], 'legacy'))
+
+- task: onebranch.pipeline.signing@1
```

Comment:
enabling OneBranch signing. Sign validation is automatically injected into the OneBranch build pipeline.